components/common/Title.tsx

```diff
@@ -8,11 +8,12 @@ interface TitleProps {
   children: React.ReactNode;
   size?: FontSizeKeys;
   centered?: boolean;
+  style?: StyleProp<ViewStyle>;
 }
 
-const Title = ({ children, size: fontSize = 'h1', centered }: TitleProps) => {
+const Title = ({ children, size: fontSize = 'h1', centered, style }: TitleProps) => {
   const textAlign = centered ? 'center' : 'left';
-  let currentStyle = styles.titleTextCustom;
+  let currentStyle = style ? style : styles.titleTextCustom;
```

Comment:
let currentStyle = [...styles.titleTextCustom, ...style] is better. We don't want to lose the default styles of the style object does not override all of the default styles.